operations/jsonnet/microservices/config.libsonnet

```diff
@@ -97,6 +97,11 @@
     backend: error 'Must specify a backend',  // gcs|s3
     bucket: error 'Must specify a bucket',
 
+    // `use_overrides_configmap` enables the tempo workloads read `overrides.yaml` from single configmap
+    // Eventually, once we enable it everywhere, we remove this and default will be true.
```

Comment:
We don't need a feature flag to roll this out gradually. I'm actually fine with making this the new default. This change doesn't cause more disruption than a normal update does (we roll all components anyway) and using a separate configmap does not have any disadvantage afaik.

Reply:
I would highly recommend using the flag, Just to avoid any surprises!. It's very straight forward and easy to cleanup once we are confident and changes are in all envs :)

Reply:
+1 to @kvrhdn 's thoughts. Let's just swap over instead of all the if/else logic in jsonnet.

Reply:
Updated. No conditional flag! :)